### ISSUE TITLE
Additional test needed: MD5 checksum validation

### ISSUE THREAD

**Issue by interob** [WFP-VAM/modape#178]

**Comment by valpesendorfer** [comment]
after #173

**Comment by interob** [comment]
@valpesendorfer is this issue still relevant / open for a reason?

**Comment by valpesendorfer** [comment]
yes, you introduced a new checksum validation method - as far as I see there are no tests implemented for it. Would be nice to have one or two
